x/dynamicfee/keeper/keeper_test.go

```diff
@@ -85,3 +86,42 @@ func TestEnabledHeight(t *testing.T) {
 		require.Equal(t, int64(10), got)
 	})
 }
+
+func TestMaxGas(t *testing.T) {
+	k, ctx := testutil.SetupKeeper(t, 0)
```

Comment:
Since `GetMaxBlockGas()` is a pure function (it does not alter state), it's not a problem here, but usually you should create a new keeper for each case to avoid state share between subtests.